src/clr-addons/components.clr-addons.scss

```diff
@@ -16,6 +16,7 @@
 @import './combobox/combobox.clarity';
 @import './date-time-container/date-time-container';
 @import './searchfield/search-field';
+@import './quick-list/add-option/add-option';
```

Comment:
you have 2 new SCSS files but only import one here

Reply:
we only use one now